src/state/pwallet/hooks/hedera.ts

```diff
@@ -553,23 +553,18 @@ export const useHederaPGLToken = (
   currencyB: Currency | undefined,
 ): [Token | undefined, Token | undefined] => {
   const chainId = useChainId();
-
-  const [pglToken, setPglToken] = useState<Token | undefined>(undefined);
   const [, pair] = usePair(currencyA, currencyB);
 
   const pairToken = pair?.liquidityToken;
 
-  const { isLoading, data } = useQuery(['get-pgl-token', pairToken?.address], fetchHederaPGLToken(pairToken, chainId));
-
-  useEffect(() => {
-    if (!isLoading && !!data) {
-      setPglToken(data);
-    }
-  }, [isLoading, data]);
+  const { data } = useQuery(
```

Comment:
lets rename this data as pglToken `{ data: pglToken }` so it will be more meaningful.